Sources/Moya/Endpoint.swift

```diff
@@ -99,7 +72,33 @@ extension Endpoint {
         request.httpMethod = method.rawValue
         request.allHTTPHeaderFields = httpHeaderFields
 
-        return try? parameterEncoding.encode(request, with: parameters)
+        switch task {
+        case .requestPlain, .uploadFile, .uploadMultipart, .downloadDestination:
+            return request
+
```

Comment:
Sure, no problem. We can talk about code style somewhere else. I've removed the empty lines.